haystack/nodes/prompt/prompt_node.py

```diff
@@ -190,6 +190,20 @@ def supports(cls, model_name_or_path: str) -> bool:
         return False
 
 
+class StopWordsCriteria(StoppingCriteria):
+    """
+    Stops text generation if any one of the stop words is generated.
+    """
+
+    def __init__(self, model_name_or_path: str, stops_words: List[str]):
+        super().__init__()
+        tokenizer = AutoTokenizer.from_pretrained(model_name_or_path)
+        self.stop_tokens = tokenizer.encode(stops_words, add_special_tokens=False, return_tensors="pt")
```

Comment:
```suggestion
    def __init__(self, model_name_or_path: str, stop_words: List[str]):
        super().__init__()
        tokenizer = AutoTokenizer.from_pretrained(model_name_or_path)
        self.stop_tokens = tokenizer.encode(stop_words, add_special_tokens=False, return_tensors="pt")
```

I'd use `stop_words` here too since it's called like that in `PromptNode.__init__()`, it might be confusing and a bit frustrating having different argument names otherwise.

Reply:
ok deal, will do!